From 966877f058d421f11e4513ac702d64c9270628c9 Mon Sep 17 00:00:00 2001
From: ulugbekbahromov <ulugbekbahromov@gmail.com>
Date: Wed, 2 Nov 2022 09:45:58 +0500
Subject: [PATCH] update 2 lines in secondFile

---
 secondFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/secondFile.txt b/secondFile.txt
index 91135c1..b14670d 100644
--- a/secondFile.txt
+++ b/secondFile.txt
@@ -1,5 +1,5 @@
-20
-21
+202
+212
 22
 23
 24
-- 
2.25.1

